docs/reference/monitoring/indices.asciidoc

```diff
@@ -28,7 +28,7 @@ and the number of replicas to two.
 ----------------------------------
 PUT /_template/custom_monitoring
```

Comment:
We should also change this request:
- use the `_index_template` endpoint 
- `order` is now `priority` (need to change it in the text above, on line 19, too)
- the `settings` object needs to be wrapped in a `template` object

Reply:
I have updated the doc as per your suggestions. please verify

Reply:
`settings` needs to be wrapped in a `template` object. The request becomes:

```
PUT /_index_template/custom_monitoring
{
  "index_patterns": [
    ".monitoring-beats-8-*",
    ".monitoring-es-8-*",
    ".monitoring-kibana-8-*",
    ".monitoring-logstash-8-*"
  ],
  "priority": 1,
  "template": {
    "settings": {
      "number_of_shards": 5,
      "number_of_replicas": 2
    }
  }
}
```

Reply:
please verify now